<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>options</class>
 <widget class="QWidget" name="options">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Mount Options</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineConfigFilePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Unlock A Cryfs/Gocryptfs/Securefs Volume With Specified Configuration File.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIdleTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatically Unmount After Specified Minutes of Inactivity Is Reached.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMountOptions">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>421</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Mount The Volume With Below Mount Options.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOK">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCancel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditIdleTime</tabstop>
  <tabstop>lineConfigFilePath</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEditMountOptions</tabstop>
  <tabstop>pbOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
